include/dem/particle_particle_contact_force.h

```diff
@@ -53,12 +54,11 @@ class ParticleParticleContactForceBase
    * candidates information for calculation of the ghost-local periodic
    * particle-particle contact forces.
    * @param dt DEM time step.
-   * @param torque Torque acting on particles.
-   * @param force Force acting on particles.
-   * @param heat_transfer_rate Heat transfer rate applied to particles.
+   * @param outcome Interaction outcomes : torque, force, heat transfer
```

Comment:
I would not write every container. If we add another one later, this comment will probably not get updated.  


```suggestion
   * @param outcome Interaction outcomes.
```